### DISCUSSION TITLE
Project Quarm/TAKP Support

### DISCUSSION THREAD

**Discussion by dhotay** [mgeitz/eqalert#290] [Ideas]
Hello, would it be feasible to add support for Project Quarm/TAKP?  The log files are saved directly in the EverQuest root folder (which would affect issue #249) and are in "eqlog_*charname*_pq.proj.txt" format.

**Comment by mgeitz** [comment]
Great question, this isn't something I've tried myself.  There may be some required manual changes I haven't considered, or rather places I just always assumed a p99 server would be used.

On start up, the parser _should_ find those logs and characters [here:](https://github.com/mgeitz/eqalert/blob/master/eqa/lib/config.py#L174-L180), regardless of server.  The only assumptions there are the directory to read from and the `eqlog_` prefix.

Any log messages unique to TAKP/Quarm not found in p99 won't be picked up by the parser as is but any new `line_type` returns in [the parser](https://github.com/mgeitz/eqalert/blob/master/eqa/lib/parser.py) will automatically be added to the config in `other.json` if you want to manually extend it for those servers.

If you are running in to any more specific errors or issues after trying eqalerts compatibility to other servers please let me know.  

I hope to add some plug-ability in the future so the parser functions can be more easily extended to generic line match files for guild specific or server specific applications #132
